shinysdr/i/webstatic/client/values.js

```diff
@@ -346,7 +346,27 @@ define([
   function isImplementing(object, interfaceName) {
     return !!object['_implements_' + interfaceName];
   }
-  exports.getInterfaces = getInterfaces;
+  exports.isImplementing = isImplementing;
+
+  // Creates a cell whose value is the array of cells in blockCell's attributes that implement interfaceName.
+  function findImplementersInBlockCell(scheduler, blockCell, interfaceName) {
+    // TODO: DerivedCell doesn't actually use its scheduler argument; why does it need to be here?
+    return new DerivedCell(anyT, scheduler, function (dirty) {
+      const block = blockCell.depend(dirty);
+      // TODO: Why doesn't depending on the block also listen on _reshapeNotice?
```

Comment:
Because the cell only reports about when its value changes, not anything recursive into that value.